signing-service-challenge/README.md

```diff
@@ -66,19 +66,19 @@ For now we need to provide two operations to our customers:
 
 Think of how to expose these operations through a RESTful HTTP based API.
 
-List / retrieval operations can optionally be implemented but aren't necessary by any means.
+`List / retrieval operations` can optionally be implemented but are not required.
 
 #### QA / Testing
 
-As we're in the area of compliance technology we need to make sure that our implementation is verifiably correct. Think of an automatable way to assure the correctness (in this challenge: adherence to the specifications) of the system.
+As we are in the business of compliance technology, we need to make sure that our implementation is verifiably compliant. Think of an automatable way to assure the correctness (in this challenge: adherence to the specifications) of the system.
 
 #### Technical Constraints & Considerations
 
 - The system will be used by many concurrent clients accessing the same resources.
-- The `signature_counter` shall be strictly monotonically increasing and ideally without any gaps.
+- The `signature_counter` has to be strictly monotonically increasing and ideally without any gaps.
 - The system currently only supports `RSA` and `ECDSA` as signature algorithms. Try to design the signing mechanism in a way that allows easy extension to other algorithms without changing the core domain logic.
-- For now it is enough to store signature devices in memory. Efficiency is not a priority for this. We might want to scale out though, therefore keep in mind that we might want to switch to a relational database at some point when designing your storage logic.
+- For now it is enough to store signature devices in memory. Efficiency is not a priority for this. In the future we might want to scale out. As you design your storage logic, keep in mind that we may later want to switch to a relational database.
 
 #### Credits
 
-This challenge is heavily influenced by the KassenSichV (Germany) as well as the RKSV (Austria) and our solutions for them.
+This challenge is heavily influenced by the [`KassenSichV`](https://kassensichv.com/) (Germany) as well as the [`RKSV`](https://www.wko.at/branchen/information-consulting/unternehmensberatung-buchhaltung-informationstechnologie/it-dienstleistung/BGBLA-2015-II-410-EN.PDF) (Austria) and our [solutions](https://developer.fiskaly.com/api) for them.
```

Comment:
I refrained from putting the links there on purpose.

Reply:
so that you know that the people can google?:)